src/main/java/org/radarcns/biovotion/BiovotionTopics.java

```diff
@@ -72,6 +75,15 @@ private BiovotionTopics() {
         gsrTopic = createTopic("android_biovotion_galvanic_skin_response",
                 BiovotionVSMGalvanicSkinResponse.getClassSchema(),
                 BiovotionVSMGalvanicSkinResponse.class);
+        accelerationTopic = createTopic("android_biovotion_acceleration",
+                BiovotionVSMAcceleration.getClassSchema(),
+                BiovotionVSMAcceleration.class);
+        ppgRawTopic = createTopic("android_biovotion_ppg_raw",
+                BiovotionVSMPhotoRaw.getClassSchema(),
+                BiovotionVSMPhotoRaw.class);
+        ledCurrentTopic = createTopic("android_biovotion_led_current",
+                BiovotionVSMLedCurrent.getClassSchema(),
+                BiovotionVSMLedCurrent.class);
```

Comment:
Could we define the topic names as a public constant? They may be reused in other projects. For instance, in MongoDb we have defined the default collection's name as the topic name. The same may apply for the Backend project, on the Kafka Stream definition we may reuse them to define topics we are going to consume.

@blootsvoets may we do the same thing also in the other device projects?